### DISCUSSION TITLE
Add teleport safe tsh support

### DISCUSSION THREAD

**Discussion by yiiman-dev** [coollabsio/coolify#3034] [Feature Requests]
We are using [Teleport ](https://goteleport.com) to safe our server networks.
Teleport is very useful for companies that wants close SSH port on all servers, but also connect to servers from anywhere with safe auth.
We closed all ssh ports and coolify dont support Teleport [tsh](https://goteleport.com/docs/connect-your-client/tsh/) CLI to connect to servers.
With adding tsh support, coolify will be safe for us.
We exposed coolify panel on our [Teleport safe panel](https://goteleport.com/docs/enroll-resources/application-access/application-access) and it is accesible from anywhere on internet, just when our devops staffs login to teleport panel.

**Comment by lucaseddev** [comment]
Did you ever figured out any workarounds for fully/safely using Teleport with coolify, with ssh ports closed?
